docs/docs/install/configuration.md

```diff
@@ -0,0 +1,34 @@
+# Configuring Mathesar 
+
+The table below shows all available environment variables supported by Mathesar. See the specific installation guides for the applicable environment variables and instructions on how to set them.
+
+> Variables marked with *Internal* should only be changed if you know what you are 
+> doing.
+
+
+### Backend Configuration
+| Name                   | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | Defaults                     |
+|------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------|
+| SECRET_KEY             | The Django secret key consists of random 50 characters strings is used to provide cryptographic signing. This key is mostly used to sign session cookies. Refer [Django docs](https://docs.djangoproject.com/en/3.2/ref/settings/#std:setting-SECRET_KEY) for more details. You can either generate a random 50 character string by yourself or you can generate a secret key using [this tool](https://djecrety.ir/)                                                                                                                               | *Required                    |
```

Comment:
The writing style is different for each of the entries, which makes it hard to get an idea of things by skimming.

Some things that stood out for me:
- For most entires, the first line provides a direct definition of what it is, while some are more descriptive.
  - `POSTGRES_PORT: The port the postgres database listens on`,
  - `SECRET_KEY: The Django secret key consists of random 50 characters strings is used to provide cryptographic signing .`
    - It would be better if it followed the same style like `SECRET_KEY: A 50 characters long random string used by Django for cryptographic signing.`
- It was hard to read the description due to the formatting of the rendered markdown. We should format it in a way to make it more readable.
   - Eg., First paragraph is the definition, a separate second paragraph for more details, third parah contains links. If there's a format for the variable that needs to be followed, it would be a separate parah.
- Some entries end with a full stop, some don't.